rfcs/0140-simple-package-paths.md

```diff
@@ -70,7 +70,7 @@ with an arbitrary `args`.
 Check the following using CI for each unit directory:
 - `pkgs.${name}` is defined as above, either automatically or with some `args` in `pkgs/top-level/all-packages.nix`.
 - `pkgs.${name}` is a [derivation](https://nixos.org/manual/nix/stable/glossary.html#gloss-derivation).
-- The `package.nix` file must not transitively refer to files outside its unit directory.
+- <a id="req-ref"> The `package.nix` file must not transitively refer to files outside its unit directory.
```

Comment:
probably doesn't matter in markdown but:

![image](https://github.com/NixOS/rfcs/assets/9866621/520f099a-c398-48e8-852c-e045615ae45c)

```suggestion
- <a id="req-ref"></a> The `package.nix` file must not transitively refer to files outside its unit directory.
```

(if this is meant to be an invisible anchor)

Reply:
Thanks, fixed!